centraldogma/src/main/java/com/linecorp/decaton/centraldogma/CentralDogmaPropertySupplier.java

```diff
@@ -103,6 +119,21 @@ public CentralDogmaPropertySupplier(CentralDogmaRepository centralDogmaRepositor
         } catch (TimeoutException e) {
             throw new RuntimeException(e);
         }
+
+        rootWatcher.watch(node -> {
+            for (PropertyDefinition<?> definition : PROPERTY_DEFINITIONS) {
+                DynamicProperty<?> p = cachedProperties.get(definition.name());
+                if (p != null && node.has(definition.name())) {
```

Comment:
🙅This is wrong. 
Actually, the PKQ-related settings should also have been updated, but this didn’t work... sorry about that.
We need to retrieve the key–value from the JSON node based on the cached property names. I’ll fix it.

Reply:
resolved